docs/pages/access-controls/access-lists/reference.mdx

```diff
@@ -0,0 +1,119 @@
+
+## What do Access Lists do?
+
+Access Lists grant roles and traits to users on a long lived basis. Users who are added
+to Access Lists and meet membership requirements will be granted these roles
+and traits when they sign into Teleport. By combining this functionality with the Access Lists'
+built in regular auditing and access reviews, Teleport administrators will have an audit trail
+for long lived access granted to users of Teleport.
+
+## How do Access Lists compare to Access Requests?
+
+Access Lists are intended for longer lived access to Teleport resources and Access Requests
+are intended for temporary elevation of privileges. Access conveyed by an Access List is
+expected to live on the order of months and access conveyed by an Access Request is intended
+to live on the order of hours or days.
+
+## Access List Ownership
+
+Access List owners are Teleport users who are granted special privileges over
+an Access List. These owners are defined explicitly as part of the Access List, and
+must be added by a Teleport user who has RBAC access to access lists, which the preset `editor`
+role has.. Owners must meet requirements in order for their ownership to be effective.
+
+Provided owners meet requirements, owners are able to do the following:
+
+- Control membership requirements.
+- List Access List members.
+- Provision and revoke membership to Access Lists.
+- Audit Access Lists.
+
+Owners are not able to add or remove owners from an Access List or control what roles
+and traits are granted by the Access List.
+
+## Access List Membership
+
+Access List members are Teleport users who are granted the roles and traits specified
+by the Access List. Upon login, users will be granted these roles and traits along with
+their statically defined user permissions. These roles and traits will then tie into
+Teleport's existing RBAC system. Members can be optionally granted an expire date, after
+which their membership will no longer confer any grants to the user.
+
+Members must meet requirements in order for their membership to be effective.
+
+## Access List Auditing
+
+Access Lists must be defined with an audit frequency, which specifies how often the
+Access List must be audited. If the Access List is not audited on time, then the Access
+List will be rendered ineffective. All members will not be granted the roles or traits
+specified by the Access List. However, owners are still able to audit the lists in order
+to re-establish the validity of the Access List.
+
+## Overview of an Access List Resource
+
+```yaml
+version: v1
+kind: access_list
+metadata:
+  name: ea6cccbe-ceac-4776-8a89-4b1365fc03f5
+spec:
+  title: "Access List Title"
+  # audit defines how frequently an Access List and its membership must be audited, along
+  # with the next audit date. If an Access List is not audited by the next audit date, the
+  # roles and traits granted by this Access List will be revoked from the users until the
+  # audit occurs.
```

Comment:
Let's remove this note for now, until we iron out what kind of behavior we want.
```suggestion
  # audit defines how frequently an Access List and its membership must be audited, along
  # with the next audit date.
```